packages/avado/.gitignore

```diff
@@ -1,7 +1,3 @@
-# ignore everything in build/ except for the Dockerfile
```

Comment:
Why did you remove this?

Reply:
The build context directory no longer contains just the `Dockerfile`, but also the configration files for Grafana and Prometheus.